### DISCUSSION TITLE
Understand the logs I'm shown when running optimize()

### DISCUSSION THREAD

**Discussion by Serendipity31** [optuna/optuna#5069] [Q&A] [answered]
The documentation surrounding logging seems to suggest that I should see a summary of how each trial went. I see something different, and I've not been able to find documentation that explains what I am seeing or how to turn it off (if I want to). 

What I see looks like this:
```{python}
Epoch 3:  |                    | 50/?  [00:04<00:00, 11.47it/s, v_num=31, train_loss=1.190]Epoch 3, global step 200: 'train_loss' reached 1.18870 (best 1.18870), saving model to 'C:\\Users\\usr\\Documents\\Repository\\lighning_logs\\version_37\\checkpoints\\epoch=3-step=200.ckpt' as top1
Epoch 4:  |                    | 50/?  [00:01<00:00, 29.77it/s, v_num=31, train_loss=1.340]Epoch 4, global step 250: 'train_loss' was not in top 1
```
In the above, I don't know where '50' comes from, or what it means that the denominator in '50/?' is a question mark. I don't know what v_num is either.

I also never see a summary of each trial. It's always on an epoch-by-epoch basis. 

Can anyone help me to understand this kind of output?

**Comment by nzw0301** [comment, marked as answer]
I believe that the log does not come from Optuna. I guess you are using pytorch lightning, if so, the log comes from it.

**Reply by Serendipity31** [reply]
@nzw0301 - Interesting idea. I'll look into that. Thanks for that. Assuming you're right, I need to figure out why I'm not seeing the per-trial summaries that the optuna docs discuss being produced by default.
